--- old
+++ new
@@ -5,1 +5,1 @@
-I want just a basic alpine server with the minimum of stuff installed. So we use the route using ```Pmbootstrap init``` to tell pmbootstrap what device we have and what options we want.
+I want just a basic alpine server with the minimum of stuff installed. So we use the route using [Pmbootstrap init](https://wiki.postmarketos.org/wiki/Installation/Using_pmbootstrap#Initializing_pmbootstrap)   to tell pmbootstrap what device we have and what options we want.
@@ -7,6 +7,2 @@
-https://wiki.postmarketos.org/wiki/Installation/Using_pmbootstrap#Initializing_pmbootstrap
-
-  https://wiki.postmarketos.org/wiki/Installation/Using_pmbootstrap#Installing_pmbootstrap
-
-On your PC, run
-
+On your PC, run 
+```$pmbootstrap init```
@@ -14,2 +10,0 @@
-$pmbootstrap init
- 