--- old
+++ new
@@ -2,1 +2,1 @@
-Immich is a popular selfhosted replacement for Goolge photos. 
+Immich is a popular selfhosted replacement for Google photos. Casting to Google chromcast devices is a hassle as the required HTTPS connection requires setting up certificates.
@@ -6,1 +6,1 @@
-This guide shows how to use CATT ( Cast All The Things) to chromecast Immich Kiosk.
+This guide shows how  to chromecast Immich Kiosk to a local device.use CATT ( Cast All The Things) to 