--- old
+++ new
@@ -2,1 +2,1 @@
-Immich is a popular selfhosted replacement for Google photos. Casting to Google chromcast devices is a hassle as the required HTTPS connection requires setting up certificates.
+Immich is a popular selfhosted replacement for Google photos. 
@@ -4,0 +4,2 @@
+Casting to Google chromcast devices is possible (https://docs.immich.app/features/casting) hassle as the required HTTPS connection requires setting up certificates.
+