--- old
+++ new
@@ -1,0 +1,1 @@
+### Bluetooth
@@ -2,0 +3,13 @@
+To set up bluetooth ;
+
+    sudo apk add bluez
+    sudo apk add dbus-broker
+    sudo rc-update add bluetooth boot
+    sudo service bluetooth start
+
+    bluetoothctl power on
+    bluetoothctl  scan le
+    bluetooth
+       scan on
+This will add bluetooth software, set it to start after a reboot, and start a scan for bluetooth low energy ( le) devices. Exit to stop.
+