--- old
+++ new
@@ -3,1 +3,29 @@
-Just follow the official guide
+Just follow the official guide for [Docker on Alpine linux]( https://wiki.alpinelinux.org/wiki/Docker).
+```
+    sudo apk upgrade
+    sudo apk update
+    sudo apk add docker
+    sudo addgroup bryan docker         
+    sudo rc-update add docker default
+    sudo service docker start
+    sudo apk add docker-cli-compose
+    sudo reboot
+```    
+    
+Obviously change 'bryan' for your user name...
+
+### Test Docker
+
+To check docker, run
+```    
+docker run hello-world 
+```
+This should pull and run a very simple test that prints a message "Hello from Docker" and then exit.
+
+### Clean up
+
+Assuming all is ok, run:
+```
+    docker system prune -a
+```
+This will clean up and remove all docker stuff that is not being used.