--- old
+++ new
@@ -12,4 +12,0 @@
-I wanted a simple static site and looked at all sorts of alternatives. My main concern was to find something simple, easy and without annoying adverts.
-
-Self hosting is an obvious way to go for 'No adverts' and I looked at  several alternatives but finally decided I didn't like the security risk of letting  everyone onto my home lab.
-
@@ -18,1 +14,1 @@
-As Github uses the Jekyll system for it's site, I at first thought it necessary to install and  understand Jekyll. But this is not so. Jekyll is I assume undoutably very flexible but seemed  very complex..too complex with all sorts of customisations and alternatives.
+Github uses the Jekyll system for it's site and I at first thought it necessary to install and  understand Jekyll. But this is not necessary. 
@@ -20,1 +16,1 @@
-Github uses Jekyll in the background. If you use a supported theme and have a few files initially setup, new posts can just be added in. 
+After setting up a repository with the special name "<yourusername>.github.io" , it only takes a couple of files to get up and running. 
@@ -22,1 +18,1 @@
-I would advise using your web browser to do the initial set up, afterwards, posts can be added using a browser or the android app.
+After that,  just add your blog posts and github automatically detects changes and rebuilds the site.
@@ -24,0 +20,2 @@
+I would advise using a normal web browser to do the initial set up. Afterwards, posts can be added using  the Github android app on a phone or tablet or a normal web browser.
+
@@ -26,1 +24,1 @@
-Your blogsite will be available at <your-user-name>.github.io. So, if you dont already have one, you might want to create your account with a friendly name; I chose BryansPlace.
+Your blogsite will be available at <your-user-name>.github.io. So, if you don't already have one, you might want to create your account with a friendly name; I chose BryansPlace.
@@ -32,1 +30,1 @@
-However, that sets up a page based site, not a blog, so we need to change the theme and set up the initial file structure
+However, that sets up a page based site, not a blog. We need to change the theme and set up the initial file structure.
@@ -41,1 +39,0 @@
-
@@ -65,2 +62,2 @@
-  My blog. This description will go into the footer 
-  or each page.
+  My blog. This description will go into the footer on each page.
+```
@@ -68,0 +65,2 @@
+The above will give you a bare bones site
+