--- old
+++ new
@@ -11,1 +11,1 @@
-Tailscale [installation](https://tailscale.com/docs/install/start) is straightforward but there are no alpine linux specific instructions
+Tailscale [installation](https://tailscale.com/docs/install/start) is straightforward but there are no alpine linux specific instructions.
@@ -13,1 +13,1 @@
-Alpine Linux has a tailscale package: https://pkgs.alpinelinux.org/packages?name=tailscale
+Alpine Linux has a tailscale package.