--- old
+++ new
@@ -6,0 +6,19 @@
+
+automation:
+  alias: "E-mail edf info"
+  description: "E-mail edf info"
+  trigger:
+    - platform: time_pattern
+      hours: "11"
+      minutes: "32"
+  condition: []
+  action:
+    - service: notify.bryansharpe50_gmail_com
+      data:
+        title: Daily EDF info
+        target: bryan@sharpe.fr
+        message:  
+          This is your EDF info: 
+ #         Tomorrow {{ ('sensor.rte_tempo_tomorrow') }} 
+ 
+ 