--- old
+++ new
@@ -3,1 +3,1 @@
-The install will have created two files that need to be flashed to the phone; lk2nd ( a qualcom secondary bootloader) and rootfs
+The install will have created two files that need to be flashed to the phone; lk2nd ( a qualcom specific secondary bootloader) and rootfs
@@ -10,6 +10,4 @@
-    pmbootstrap flasher flash_lk2nd --boot_a
-    pmbootstrap flasher flash_lk2nd --boot_b
-    fastboot erase system_a
-    fastboot erase system_b
-    pmbootstrap flasher flash_rootfs
-    fastboot reboot 
+    $ pmbootstrap flasher flash_lk2nd --partition boot_a
+    $ pmbootstrap flasher flash_lk2nd --partition boot_b
+    $ pmbootstrap flasher flash_rootfs --partition userdata
+    $ fastboot reboot 
@@ -17,0 +15,1 @@
+The system should reboot. As the display option was none, there is little indication that things have succeded
@@ -18,1 +17,0 @@
-