--- old
+++ new
@@ -9,1 +9,1 @@
-## Repurposing the Past, Powering the Future
+## Repurposing old smartphones
@@ -11,1 +11,1 @@
-Welcome to my corner of the web. I like to breath new life into "obsolete" hardware—specifically turning old smartphones into functional, low-power homelabs.
+Welcome to my corner of the web. I like to breath new life into "obsolete" phones. Specifically turning them into functional, low-power homelabs.
@@ -13,2 +13,2 @@
-### What I Do
-Most smartphones are retired long before their hardware is actually useless. By utilizing **PostmarketOS** (based on **Alpine Linux**), old abandoned Android devices can be used as  stable, lightweight servers.
+### PostmarketOS
+After looking at loads of alternatives, I homed in on PostmarketOS. Although it is a system Most smartphones are retired long before their hardware is actually useless. By utilizing **PostmarketOS** (based on **Alpine Linux**), old abandoned Android devices can be used as  stable, lightweight servers.