--- old
+++ new
@@ -5,0 +5,2 @@
+This site contains stuff I want to document in case I have to do it again. 
+Hopefully also useful to others.
@@ -6,1 +8,1 @@
-## Welcome to my corner of the web
+For over three years now, I have run my homelab server(s) on old phones (two Xiaomi A1 devices). By installing PostmarketOS you get a simple Alpine linux  system which is, importantly for me, able to host docker containers.  
@@ -8,0 +10,1 @@
+**PostmarketOS**  installs without rooting the phone via its installer progran, Pmbootstrap. 
@@ -9,5 +12,0 @@
-
-For over three years now, I have run my homelab server(s) on old phones. By installing PostmarketOS you get a simple Alpine linux  system which is, importantly for me, able to host docker containers.  
-
-**PostmarketOS**  installs without rooting the phone via its installer progran, Pmbootstrap. I have two Xiaomi A1 devices that have been running for over three years with no problems.
-