--- old
+++ new
@@ -67,1 +67,1 @@
-USB debugging. %HEADING%usb-debugging-%HEADING%
+#### USB debugging.
@@ -76,1 +76,1 @@
-USB cable %HEADING%usb-cable%HEADING%
+#### USB cable 
@@ -80,1 +80,1 @@
-Flash mode %HEADING%flash-mode%HEADING%
+#### Flash mode
@@ -82,1 +82,1 @@
-Thr phone needs to be put into flash mode.
+Thr phone needs to be put into 'flash mode'.
@@ -126,0 +126,1 @@
+### Install
@@ -127,0 +128,1 @@
+The command ```pmbootstrap install``` installs the chosen options into the image files that will be flashed to the phone.
@@ -128,0 +130,1 @@
+It does not install anything onto the phone!
@@ -129,0 +132,34 @@
+It will ask for a password that will be needed to log into the phone later.
+
+### Flash to phone
+
+The install will have created two files that need to be flashed to the phone; lk2nd ( a qualcom specific secondary bootloader) and rootfs ( the root file system).
+
+The Xiaomi A1 tissot phone actually has what they call an A/B structure. I won’t explain it here, but I want to be sure it always boots into the new system, so I choose to flash it to both A and B slots
+
+Ensure device is still in fastboot mode, and plugged in via USB. Then, on the PC, run:
+```
+    $ pmbootstrap flasher flash_lk2nd --partition boot_a
+    $ pmbootstrap flasher flash_lk2nd --partition boot_b
+    $ pmbootstrap flasher flash_rootfs --partition userdata
+    $ fastboot reboot 
+```    
+Wait a while as the phone reboots into PostmarketOS.
+
+As the display option was none, there is little indication that things have succeded.
+
+### Login thru SSH
+With the USB cable still connected, from your PC terminal ; with your user name run
+```
+    $ssh user@172.16.42.1
+```     
+You will be asked for your password.
+
+Hopefully, you now have control of the phone from your computer.
+
+
+
+
+
+
+